File: Facebook.NET/GraphAPI.Pagination.Cursor.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        public sealed partial class Pagination
        {
            [DataContract]
            public sealed class Cursor
            {
                [DataMember(Name = "after")]
                public string After { get; set; }
                
                [DataMember(Name = "before")]
                public string Before { get; set; }
            }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Pagination.Cursor.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        Partial Public NotInheritable Class Pagination
            <DataContract()>
            Public NotInheritable Class Cursor
                <DataMember(Name:="after")>
                Public Property After() As String
                <DataMember(Name:="before")>
                Public Property Before() As String
            End Class
        End Class
    End Class
End Namespace
